app/assets/stylesheets/custom.scss

```diff
@@ -208,3 +209,16 @@ th {
 th {
   text-align: center;
 }
+
+li.active > a
+{
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Properties should be ordered background-color, color, outline, text-decoration